apps/console/src/features/private-key-jwt/pages/private-key-jwt-config.tsx

```diff
@@ -60,19 +59,19 @@ export const PrivateKeyJWTConfig: FunctionComponent<PrivateKeyJWTConfigPageInter
         history.push(AppConstants.getPaths().get("PRIVATE_KEY_JWT_CONFIG_EDIT"));
     };
 
-    return ( tokenReuseData ?
-        (<Section
-            data-componentid={ `${componentId}-settings-section` }
-            description={ t("console:manage.features.jwtPrivateKeyConfiguration.description") }
-            icon={ getSettingsSectionIcons().jwtPrivateKeyValidation }
-            header={ t(
-                "console:manage.features.jwtPrivateKeyConfiguration.pageTitle" 
-            ) }
-            onPrimaryActionClick={ handleSelection }
-            primaryAction={ "Configure" }
-        >
-            <Divider hidden/>
-        </Section>)
+    return ( tokenReuseData
+        ? (
+            <Section
+                data-componentid={ `${componentId}-settings-section` }
+                description={ t("console:manage.features.jwtPrivateKeyConfiguration.description") }
+                icon={ getSettingsSectionIcons().jwtPrivateKeyValidation }
+                header={ t(
+                    "console:manage.features.jwtPrivateKeyConfiguration.pageTitle" 
+                ) }
+                onPrimaryActionClick={ handleSelection }
+                primaryAction={ "Configure" }
```

Comment:
Shall we i18n this string :)

Reply:
fixed https://github.com/wso2/identity-apps/pull/4297/commits/547ecfea29e748359a3c27617f2eccfa4a6d2418